### PR TITLE
detect-file-data/v4: add tests for SMTP file data

### PR DESCRIPTION
**Pull request by TheKharleeci**
Test conversion is shown below:

smtp-file-data-01: DetectEngineSMTPFiledataTest01

smtp-file-data-02: DetectEngineSMTPFiledataTest03

Previous PR: #644 

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4938

### CODE REVIEW

File: tests/smtp-file-data/smtp-file-data-02/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Description
+
+Test file_data keyword against smtp for fragmented pcaps
```
Comment (inashivb) [inline tests/smtp-file-data/smtp-file-data-02/README.md:3]: nit: fragmented data

**Review by inashivb — CHANGES_REQUESTED** [review]
Just one nit as of now. Just approved the builds, let's see if it passes.
Rest, LGTM! Good job! :)

**Comment by jufajardini** [comment]
> Just one nit as of now. Just approved the builds, let's see if it passes. Rest, LGTM! Good job! :)

We are still getting failures for 6.0.x (and 5.0.x), but I see that the backport which we expected would fix this failure has been merged a few days ago...

**Comment by inashivb** [comment]
> > Just one nit as of now. Just approved the builds, let's see if it passes. Rest, LGTM! Good job! :)
> 
> We are still getting failures for 6.0.x (and 5.0.x), but I see that the backport which we expected would fix this failure has been merged a few days ago...

Seems like some commit is still missing that affects the logged file size.

**Comment by TheKharleeci** [comment]
Followed by: #708

**Pull request closed without merging**